modules/javascript/partytown-web-worker/load.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Module Name: Partytown Web Worker
+ * Description: Run JavaScript in a separate Web Worker with the help of Partytown library.
+ * Experimental: Yes
+ *
+ * @since   1.0.0
```

Comment:
I believe all of the @since annotations can use n.e.x.t to be transformed to the proper version on release.

Reply:
Updated